src/Aspire.Hosting.Keycloak/api/Aspire.Hosting.Keycloak.cs

```diff
@@ -16,7 +16,7 @@ public static partial class KeycloakResourceBuilderExtensions
 
         public static ApplicationModel.IResourceBuilder<ApplicationModel.KeycloakResource> WithDataVolume(this ApplicationModel.IResourceBuilder<ApplicationModel.KeycloakResource> builder, string? name = null) { throw null; }
 
-        public static ApplicationModel.IResourceBuilder<ApplicationModel.KeycloakResource> WithRealmImport(this ApplicationModel.IResourceBuilder<ApplicationModel.KeycloakResource> builder, string importDirectory, bool isReadOnly = false) { throw null; }
+        public static ApplicationModel.IResourceBuilder<ApplicationModel.KeycloakResource> WithRealmImport(this ApplicationModel.IResourceBuilder<ApplicationModel.KeycloakResource> builder, string import, bool isReadOnly = false) { throw null; }
```

Comment:
still wrong in api file
https://github.com/dotnet/aspire/blob/1be1ad5b0b4f65138882f8481162e7216b08dc4b/src/Aspire.Hosting.Keycloak/api/Aspire.Hosting.Keycloak.cs#L19
does it matter / should these be automatically regenereated?

Reply:
@danmoseley - I'm not understanding. the api file that you are pointing to above represents what we shipped in 9.0. In 9.1 we changed the parameter name, see [Refactor WithRealmImport to accept file or directory (dotnet/aspire#7121)](https://github.com/dotnet/aspire/pull/7121)

Note that this whole package is still in preview, so we can adjust the APIs.